atest/test/02_Content_Keywords/assertions.robot

```diff
@@ -45,15 +45,51 @@ InEqual
 
 Greater Than
     Get Style    //*[@id="progress"]    width    >    100
```

Comment:
You could change this too?

Reply:
It was three times xpath.

now it is xpath, css and id

thats why i did not change all